### PR TITLE
testing/py-tox && testing/py-pluggy: new aports

### PR DESCRIPTION
**Pull request by itoffshore**
* py-tox
* py-pluggy

tests pass in my local repo - temporaily disabled for travis

### CODE REVIEW

File: testing/py-pluggy/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	depends="$depends $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+check() {
+        cd "$builddir"
+	sed -i 's|envlist.*|envlist=check,py{27,36}-pytest{30}|' tox.ini
```
Comment (Shizmob) [inline testing/py-pluggy/APKBUILD]: There should be a tab here.

File: testing/py-tox/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+pkgver=2.7.0
+pkgrel=0
+pkgdesc="virtualenv management and test command line tool"
+url="https://tox.readthedocs.org/"
+arch="noarch"
+license="MIT"
+_depends="py-virtualenv"
+depends="py-py py-pluggy py-argparse"
+makedepends="python2-dev python3-dev py-setuptools"
+options="!check" # temp disable as requires itself
```
Comment (Shizmob) [inline testing/py-tox/APKBUILD]: Is there no way to set `PYTHONPATH` to circumvent this?

File: testing/py-pluggy/APKBUILD (hunk 1 above)
Comment (itoffshore) [inline testing/py-pluggy/APKBUILD]: fixed

File: testing/py-tox/APKBUILD (hunk 2 above)
Comment (itoffshore) [inline testing/py-tox/APKBUILD]: the `tox` binary does not yet exist when `check()` runs:

```
>>> py-tox: Entering fakeroot...
/usr/bin/abuild: line 2356: tox: not found
>>> ERROR: py-tox*: check failed
>>> ERROR: py-tox: all failed
>>> py-tox: Uninstalling dependencies...
musl64 [~/aports/testing/py-tox]$ find pkg -type f -name tox
find: ‘pkg’: No such file or directory
musl64 [~/aports/testing/py-tox]$ find src -type f -name tox
musl64 [~/aports/testing/py-tox]$ 
```

the tests need to be enabled after `py-tox` is in the repos.

**Comment by itoffshore** [comment]
`py-tox` tests passing in my local repo:

```
===================================================================== 289 tests deselected =====================================================================
========================================================== 18 passed, 289 deselected in 1.45 seconds ===========================================================
___________________________________________________________________________ summary ____________________________________________________________________________
  check: commands succeeded
  py27-pytest30: commands succeeded
  py36-pytest30: commands succeeded
  flakes: commands succeeded
  congratulations :)
```

**Pull request closed without merging**